frontend/public/components/factory/table.tsx

```diff
@@ -3,6 +3,7 @@ import * as PropTypes from 'prop-types';
 import * as React from 'react';
 import { connect } from 'react-redux';
 import { getNodeRoles, getMachinePhase } from '@console/shared';
+import { getSortableOperandStatus } from '@console/operator-lifecycle-manager/src/components/operand';
```

Comment:
I think we don't want `frontend/public/components` to import from a `frontend/packages/...` dir.
This could create some circular dependency issues/leaky abstractions.

Reply:
+1 core console shouldn't import from static plugins

Reply:
The right fix would be to add an extension point that allows plugins to contribute a table sort function.

@christianvogt I vaguely recall we had to do something like this before. Do you remember?

Reply:
Ya this looks like a bad code dependency.

Sorting sounds like something we should be able to pass in as a prop to the table.
Otherwise an extension works too.

Reply:
I was expecting a prop to the table as well.